setup_cluster.sh

```diff
@@ -201,7 +202,7 @@ function remove_self_provisioner_role() {
 }
 
 # https://docs.openshift.com/container-platform/4.4/applications/pruning-objects.html
-func enabe_imagepruner() {
+function enabe_imagepruner() {
```

Comment:
was it intentional to have this as func instead of function?

Reply:
I don't think so.

Reply:
i'm also seeing this error: `The Subscription "serverless-operator" is invalid: spec.channel: Invalid value: "number": spec.channel in body must be of type string: "number"` 

looking here: https://github.com/codeready-toolchain/toolchain-infra/blob/0f1a83db8d4352257c17c0abbece96b03f30fdd1/config/operators/serverless/serving_subscription.yaml#L15 it appears to be a number instead of a string - would you like me to change it to a string?

Reply:
Yeah it should be a string. So, yes, please fix it too.

Reply:
fixed here: 9cdcd2aa1fbd8bca282487755cca21de5cad11f4